primitives/ethereum/src/pow.rs

```diff
@@ -105,6 +110,7 @@ impl EthashPartial {
 				m.insert(4370000, 3000000);
 				m.insert(7280000, 2000000);
 				m.insert(0x8c6180, 0x3d0900);
+				m.insert(0xc3d0e8, 0xaae60);
```

Comment:
This looks not right, compared to https://github.com/openethereum/openethereum/pull/561/files#diff-06bc3942f66cb315836147192afb41985b91ee270161b398f69a2d07fd4ddc05R141

Reply:
I forgot where I got these values.
Should we update this to the latest openethereum code in the next runtime upgrade?

Reply:
Yes, I double checked with geth, openethereum's code should be right.

Reply:
Should this be included in darwinia-v0.11.7?
I'll fire a PR right now.

Reply:
OK, darwinia-v0.11.7 branch is a good place, and don't forget we also need it in master.